pkg/backend/backend.go

```diff
@@ -82,6 +98,14 @@ type Backend interface {
 	// URL returns a URL at which information about this backend may be seen.
 	URL() string
 
+	// ParsePolicyPackReference takes a fully-qualified PolicyPack name (i.e., of the form
```

Comment:
Could we instead put these on the httpstate Backend type directly?

Reply:
Yeah.